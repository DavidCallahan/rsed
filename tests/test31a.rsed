counter="cnt"
foreach all
    if $($counter) == "" then
       $($counter) = 1
    else
       $($counter) = $($counter) + 1
    end
    count=$($counter)
    print "$count: $CURRENT"
end
