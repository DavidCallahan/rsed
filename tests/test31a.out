1: 1
2: 2
3: 3 more
4: 4
5: 5 less
6: 6
